db/blob/blob_file_builder_test.cc

```diff
@@ -122,10 +122,8 @@ TEST_F(BlobFileBuilderTest, BuildAndCheckOneFile) {
   constexpr size_t value_offset = 1234;
 
   Options options;
-  options.cf_paths.emplace_back(
-      test::PerThreadDBPath(&mock_env_,
-                            "BlobFileBuilderTest_BuildAndCheckOneFile"),
-      0);
+  options.blob_path = test::PerThreadDBPath(
+      &mock_env_, "BlobFileBuilderTest_BuildAndCheckOneFile");
```

Comment:
Is this change necessary/expected?  Before this change, both Blob and SST files would be written to this test directory.  After the change, Blob and SST files will be written to separate places.  I agree this change is not likely to cause problems, but am wondering if there might be unintended side effects.


Note that this same change is applied to many tests...